doc/source/how-to-save-and-load-model-checkpoints.rst

```diff
@@ -21,35 +21,42 @@ returns those aggregated weights to the caller (i.e., the server):
         def aggregate_fit(
             self,
             server_round: int,
-            results: List[Tuple[fl.server.client_proxy.ClientProxy, fl.common.FitRes]],
-            failures: List[Union[Tuple[ClientProxy, FitRes], BaseException]],
-        ) -> Tuple[Optional[Parameters], Dict[str, Scalar]]:
+            results: list[tuple[fl.server.client_proxy.ClientProxy, fl.common.FitRes]],
+            failures: list[Union[tuple[ClientProxy, FitRes], BaseException]],
+        ) -> tuple[Optional[Parameters], dict[str, Scalar]]:
 
             # Call aggregate_fit from base class (FedAvg) to aggregate parameters and metrics
             aggregated_parameters, aggregated_metrics = super().aggregate_fit(
                 server_round, results, failures
             )
 
             if aggregated_parameters is not None:
-                # Convert `Parameters` to `List[np.ndarray]`
-                aggregated_ndarrays: List[np.ndarray] = fl.common.parameters_to_ndarrays(
+                # Convert `Parameters` to `list[np.ndarray]`
+                aggregated_ndarrays: list[np.ndarray] = fl.common.parameters_to_ndarrays(
                     aggregated_parameters
                 )
 
-                # Save aggregated_ndarrays
+                # Save aggregated_ndarrays to disk
                 print(f"Saving round {server_round} aggregated_ndarrays...")
                 np.savez(f"round-{server_round}-weights.npz", *aggregated_ndarrays)
 
             return aggregated_parameters, aggregated_metrics
 
 
-    # Create strategy and run server
-    strategy = SaveModelStrategy(
-        # (same arguments as FedAvg here)
-    )
-    fl.server.start_server(strategy=strategy)
+    # Create strategy and pass into ServerApp
+    def server_fn(context):
+        strategy = SaveModelStrategy(
+            # (same arguments as FedAvg here)
+        )
+        config = ServerConfig(
+            # (your server configs here, like `num-server-rounds`)
+        )
```

Comment:
```suggestion
        config = ServerConfig(num_rounds=3)
```

Reply:
`num-server-rounds` is not a key in `ServerConfig`. I'm thinking showing it like i propose above keeps things simpler and easier for others to copy right away and use or modify.